compiler/plugins/target/CUDA/CUDATarget.cpp

```diff
@@ -66,37 +66,33 @@ struct CUDAOptions {
     binder.opt<bool>("iree-hal-cuda-dump-ptx", dumpPtx, llvm::cl::cat(category),
                      llvm::cl::desc("Dump ptx to the debug stream."));
 
-    binder.opt<std::string>("iree-hal-cuda-llvm-target-arch", clTargetChip,
+    binder.opt<std::string>("iree-cuda-target", clTarget,
```

Comment:
> This is a breaking change.

We could keep the old flags as deprecated to give some time to transition. If the functionality is the same, just forward the old values into the new flags.

Reply:
I'm leaning towards to just force the break immedidately. It's gonna happen anyway and people typically don't change until stuff break. :)

Reply:
Let's be careful with the sequencing and not just break compatibility because we can.

@benvanik has some breaking changes pending too, so we should at least try to batch those together as part of a stable release.

Reply:
SG. I'm happy to wait until the next train of breaking changes is ready. :D